### ISSUE TITLE
TimeDistributed

### ISSUE THREAD

**Issue by korosig** [josephazar/MLSTM-DenseNet#1]
Hi there,
Nice GitRepo :-)

Could you tell me why are You using comment in these lines:

    #x=Permute((3,1,2))(x)
    #x=TimeDistributed(Flatten())(x)
    #x = Permute((2, 1))(x)

in netflow_densenet1d.py

**Comment by josephazar** [comment]
Hello, 

I was trying a densenet-LSTM model first. The first two commented line are related and they should be like this:

    x =BatchNormalization(gamma_regularizer=l2(weight_decay),beta_regularizer=l2(weight_decay))(x)
    x = Activation('relu')(x)
    
    x=Permute((3,1,2))(x)
    x=TimeDistributed(Flatten())(x)
    x=LSTM(32)(x)

    x = Dense(nb_classes, activation='softmax')(x)

While the final commented line (x = Permute((2, 1))(x)) is not related to the lines above. As I remember, when the number of features is greater than the number of timesteps I used to do the permutation to test whether this will impact the outcomes.

I'd change the code, and delete those lines to prevent ambiguity. Thank you for that comment.
